azurerm/internal/services/databricks/databricks_workspace_resource_test.go

```diff
@@ -50,6 +46,84 @@ func TestAccDatabricksWorkspace_requiresImport(t *testing.T) {
 	})
 }
 
+func TestAccDatabricksWorkspace_machineLearning(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_databricks_workspace", "test")
+	r := DatabricksWorkspaceResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.machineLearning(data, "standard"),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+// This is currently untestable due to https://github.com/Azure/azure-rest-api-specs/issues/8075
+// Per jlichwa: It is as designed, purge protection is irreversible. Otherwise hacker would be able
+// to delete your key vault without option to recover. #please-close
+// Unable to delete key vault since purge protection is required by the resource:
+// Code="StorageAccountOperationFailed" Message="Unable to update storage account 'dbstoragel2bko37m6ebog': 'Keyvault policy recoverable is not set'."
+// If I Add Purge Protection to be true the clean up gets the below error:
+// keyvault.BaseClient#PurgeDeletedKey: Failure responding to request: StatusCode=403 -- Original Error: autorest/azure: Service returned an error.
+// Status=403 Code="Forbidden" Message="The user, group or application 'appid=XXXX;oid=XXXX;iss=https://sts.windows.net/XXXX/' does not have keys
+// purge permission on key vault 'acctest-kv-rczeo;location=westeurope'. InnerError={"code":"ForbiddenByPolicy"}
```

Comment:
we need to test this resource so can we please see about the service removing this restriction? it doesn't make sense to enforce this

Reply:
We have tried, it is a hard no from the service team so I am really stuck.